integration-tests/tests/workspace-dashboard.js

```diff
@@ -1,6 +1,6 @@
 // This test is owned by the Workspaces Team.
 const _ = require('lodash/fp')
-const { overrideConfig, viewWorkspaceDashboard, withWorkspace } = require('../utils/integration-helpers')
```

Comment:
@jdcanas please also delete `overrideConfig`, as both usages are removed. The method as-is has a problem because it does not wait for the spinner (@aweng98 was actually in the process of investigating this). I don't want us to leave an unused method sitting around that could potentially cause a problem for someone else in the future.

Reply:
Ping again on ^

Reply:
Yeah, this is on hold until to official release date and bug bash, so focusing on that currently :). Fixing this now though...